### DISCUSSION TITLE
When a formula does not work try adding a space 🤷

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/sa4ss#112] [Show and tell]
Recently, @shcaba and I went through the painful process of trying to remove unicode from the assessment .Rmd files only to run into problems. In the biology section, there was text that looked like δ18O in the .Rmd file. This led to an "unrecognized character" in the tagging process, which was a pain to deal with. So, we tried to fix the .Rmd file but `($\delta$180)` did not work but `( $\delta$ 180)` did. I just want to highlight that the translation from .Rmd to .md to .tex can be finicky but I encourage authors to stick with it. If you are having problems with a particular chunk of LaTeX, I encourage you to navigate to [quicklatex.com](https://www.quicklatex.com/) where you can quickly see what `$\delta$` looks like without needing to compile a document. 

So, thanks to @shcaba for bearing with me and putting in the time to figure this out! You can see the commit that led to the fixes [here](https://github.com/shcaba/Sebastes_melanops_OR/commit/8c7a3163d5a5a6886ce92e2447be15218cab2c74#diff-f74f38f898b75004d496d051cdf4e7068d35ea9ae00417096f696fc918d43668).

**Comment by shcaba** [comment]
Thank you for helping with this. Adding the space (e.g., $\delta$ 180) seemed to fix things.

**Comment by iantaylor-NOAA** [comment]
I had a similar issue with a `ΔAIC` which took a long time to find. In this case it was OK with no space: `$\Delta$AIC`.
I started posting this issue comment to ask if you had tips for finding unicode characters, but then I thought to ask the internet instead and found this easy to follow solution: using regular expression search for `[^\x00-\x7f]` (via https://weblogs.asp.net/kon/finding-those-pesky-unicode-characters-in-visual-studio).

That search also turned up various characters that didn't cause any issues with compiling the PDF: `“”` (which maybe should be changed anyway), as well as `ñ`, `°`, `…`, and `–` (en-dash).
